main.js

```diff
@@ -0,0 +1,203 @@
+url = 'https://raw.githubusercontent.com/alexsimkovich/patronage/main/api/data.json';
+
+let inCartItems = 0; //ile jest produktów w koszyku
+let productsInCart = []; //tablica do zapisywania ile jest danego produktu w koszyku
+let totalCost = 0; //koszt całkowity zamówienia
+
+
+fetch(url)
+    .then(function (response){
+        return response.json();
+    })
+    .then(function (data){
+        appendData(data);
+    })
+    .catch(function (err){
+        console.log(err)
+    });
+
+
+function appendData(data){ //załadowanie danych na stronie
+

```

Comment:
można to usprawnić, np. użyć https://developer.mozilla.org/pl/docs/Web/API/Element/insertAdjacentHTML

```
function appendData(data) {
    const column1 = document.getElementById("col1");

    data.forEach(item => {
        productsInCart.push({ 'inCart': 0 })
        column1.insertAdjacentHTML('afterbegin', `
          <div class='product'>
            <img src='${item.image}' class='product-image' />
            <h3 class='title-product'>${item.title}</h3>
            <h3 class='price-product'>${item.price.toFixed(2)}</h3>
            <h3 class='ingredients-product'>${item.ingredients.join(', ')}</h3>
            <a class='add-cart cart-button'>Zamów</a>
          </div>
    `)
    })
}
```